src/typography/expandable-text.spec.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { render } from 'enzyme'
+import ExpandableText from 'govuk-react-components/typography/expandable-text';
+
+describe('The Expandable Text Component', () => {
+
+  it('should render correctly', () => {
+    expect(
+      render(<ExpandableText title="Lorem ipsum">Dolor set amet</ExpandableText>)
+    ).toMatchSnapshot()
+  });
+
```

Comment:
Probably worth adding three additional tests here:

1. Passing components in as the 'children' prop
2. Passing components in as the 'title' prop
3. Passing in the `open` prop I mentioned above